src/compas_robots/resources/basic.py

```diff
@@ -165,7 +165,10 @@ def __init__(self, path, support_package):
         super(LocalPackageMeshLoader, self).__init__()
         self.path = path
         self.support_package = support_package
-        self.schema_prefix = "package://" + self.support_package + "/"
+        if support_package is None or support_package == "":
```

Comment:
The pythonic way is to rely on truthy testing:
```suggestion
        if support_package:
```